manticore/core/smtlib/solver.py

```diff
@@ -261,6 +305,8 @@ def is_started(self):
 
 
 class SMTLIBSolver(Solver):
+    ncores: Optional[int] = None
```

Comment:
Ahh. Here!  Why the user  of an SMTLIBSolver needs know about the "cores"?

Reply:
That's necessary to know how much workers create. Originally, every worker used one cpu, but now the portofolio solver can use many of them.